docs/current/guides/snippets/build-test-publish-php/dagger.php

```diff
@@ -0,0 +1,277 @@
+<?php
+// include auto-loader
+include 'vendor/autoload.php';
```

Comment:
Should this be `require_once __DIR__ . '/vendor/autoload.php'` instead?

Reply:
Added